llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -882,6 +882,8 @@ class VPInstruction : public VPRecipeWithIRFlags,
     // Extracts the first active lane of a vector, where the first operand is
     // the predicate, and the second operand is the vector to extract.
     ExtractFirstActive,
+    // Calculates the first active lane index of the vector predicate operand.
+    FirstActiveLane,
```

Comment:
Do we need both `ExtractFirstActive` and `FirstActiveLane` or can we use FirstActiveLane with a regular extract VPInstruction?

Reply:
Yep, I can do that. Split off into https://github.com/llvm/llvm-project/pull/131118.